contracts/chain-adapters/Arbitrum_CustomGasToken_Adapter.sol

```diff
@@ -296,4 +311,12 @@ contract Arbitrum_CustomGasToken_Adapter is AdapterInterface, CircleCCTPAdapter
         if (CUSTOM_GAS_TOKEN.balanceOf(address(this)) < requiredL1CallValue) revert InsufficientCustomGasToken();
         return requiredL1CallValue;
     }
+
+    function _from18ToNativeDecimals(uint256 amount) internal view returns (uint256) {
+        uint8 nativeTokenDecimals = L1_INBOX.bridge().nativeTokenDecimals();
+        if (nativeTokenDecimals < 18) {
+            return amount / 10**(18 - nativeTokenDecimals);
```

Comment:
Maybe I'm missing something, but it seems like we will return 0 for all tokens which have decimals < 12? Our L2 gasLimit is 2_000_000, so for example, if `getL1CallValue()` is called and the nativeTokenDecimals() is, for example, 6, this will just be (2_000_000/ 10**12) = 0?